components/CellIDishID.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import {
+  Image,
+  Platform,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  FlatList,
+  CheckBox,
+  TextInput,
+} from "react-native-web";
+import styled from "styled-components";
+import { ButtonPlus, ButtonMinus } from "./PlusMinusButton";
+
+export default function CellItemDetail({ item, navigation }) {
+  // console.log(item)
+  return (
+    <React.Fragment>
+      <View style={styles.subtitleContainer}>
+        <Text style={styles.subtitle}>Adds</Text>
+      </View>
+      <View style={styles.container}>
+        <View style={styles.checkboxContainer}>
+          <CheckBox value={false} style={styles.checkbox} />
+          <Text style={styles.label}>Cheese Cheese</Text>
+          <Text style={styles.labelPrice}>+$2,50</Text>
+        </View>
+        <View style={styles.checkboxContainer}>
+          <CheckBox value={false} style={styles.checkbox} />
+          <Text style={styles.label}>Bacon</Text>
+          <Text style={styles.labelPrice}>+$2,50</Text>
+        </View>
+      </View>
+      <View style={{ padding: 10 }}>
+        <Text>Special Instructions</Text>
+        <View style={styles.textInputView}>
+          <TextInput
+            placeholder="Dressing on the side? No pickles? Let us know..."
+            underlineColorAndroid="transparent"
+            style={styles.textInput}
+            multiline={true}
+            numberOfLines={1}
+            onChangeText={(text) => console.log(text)}
+          />
+        </View>
+      </View>
+    </React.Fragment>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    borderBottomWidth: 1,
+    borderColor: "#F3F3F3",
+    paddingVertical: 10,
+    paddingHorizontal: 20,
+    flex: 1,
+    // flexDirection: "row",
+    // borderRadius: 5
+    // justifyContent: 'space-evenly'
+  },
```

Comment:
on keep those comments if you gonna use them later but just delete if we don't need